doc/ref_model/chapters/appendix-a.md

```diff
@@ -49,7 +49,7 @@ Current CNTT Policies:
 
 **SR-IOV:**
 
-CNTT recognizes that today, SR-IOV provides a critical Capability for increased throughput over network interfaces at an economical cost. As such, the CNTT approach to SR-IOV is to (detailed policy language under development; to be tied in with VNF Evolution).
+CNTT recognizes that today, SR-IOV provides a critical Capability for increased throughput over network interfaces at an economical cost. As such, the CNTT approach to SR-IOV is to (THIS PR WILL PROVIDE PROPOSED COMPROMISE LANGUAGE FOR SRIOV).
```

Comment:
a critical Capability for increased -> a mechanism to increase

Reply:
@rabi-abdel There were structural changes to that sentence, but I kept your intent in mind. If the changes address your concern, pls Resolve this sub-thread. Thanks, -Mark